packages/action-sheet/src/action-sheet-item.tsx

```diff
@@ -82,8 +82,7 @@ export const ActionSheetItem = ({
   const { onClose } = useActionSheet()
 
   const handleClick = () => {
-    onClick?.()
-    onClose?.()
+    onClick ? !onClick() && onClose && onClose() : onClose && onClose()
```

Comment:
```suggestion
    onClick ? !onClick() && onClose?.() : onClose?.()
```
똑같긴 한데 이렇게 줄여쓰는것도 가능하겠네요!

Reply:
수정했습니다!